dataprofiler/labelers/data_processing.py

```diff
@@ -1231,6 +1232,9 @@ def __init__(self, default_label='UNKNOWN', pad_label='PAD',
         :param flatten_separator: separator used to put between flattened
             samples.
         :type flatten_separator: str
+        :param is_pred_labels: (default: true) if true, will convert the model
+            indexes to the label strings given the label_mapping
```

Comment:
nit-pick, but indices not indexes.

Reply:
https://stackoverflow.com/questions/1378781/proper-terminology-should-i-say-indexes-or-indices